samples/polars/noxfile.py

```diff
@@ -0,0 +1,292 @@
+# Copyright 2019 Google LLC
```

Comment:
Is this file auto generated from a template? If not, can we update the copyright year to `2024`? Also check other new files too?

Reply:
This is templated, so not much we can do.